pkg/conversions/gatewayroute_types_conversion.go

```diff
@@ -39,11 +39,51 @@ func Convert_CRD_GRPCGatewayRouteAction_To_SDK_GrpcGatewayRouteAction(crdObj *ap
 	if err := Convert_CRD_GatewayRouteTarget_To_SDK_GatewayRouteTarget(&crdObj.Target, sdkObj.Target, scope); err != nil {
 		return err
 	}
+
+	if crdObj.Rewrite != nil {
+		Convert_CRD_GRPCHostnameRewrite_To_SDK_GrpcHostnameRewrite(crdObj.Rewrite, sdkObj.Rewrite)
+	}
 	return nil
 }
 
+func Convert_CRD_GRPCHostnameRewrite_To_SDK_GrpcHostnameRewrite(crdObj *appmesh.GrpcGatewayRouteRewrite, sdkObj *appmeshsdk.GrpcGatewayRouteRewrite) {
+	// TODO : Need to check with AppMesh API team
```

Comment:
Can we remove this now?

Reply:
Please remove these comments in the final diff

Reply:
Yes, doing that